packages/app/components/drop/drop-form.tsx

```diff
@@ -281,6 +282,25 @@ export const DropForm = () => {
     );
   }
 
+  const handleFileChange = (file: string | File) => {
+    let extension;
+    // On Native file is a string uri
+    if (typeof file === "string") {
+      extension = file.split(".").pop();
```

Comment:
if using MIME types to get it will be better because some files haven't the extension.

Reply:
yeah, we use MIME type on web. On native, I don't think there's a way yet. https://docs.expo.dev/versions/latest/sdk/imagepicker/#imageinfo